doc/nrf/releases/release-notes-latest.rst

```diff
@@ -252,19 +252,19 @@ Zephyr
 
 .. NOTE TO MAINTAINERS: The latest Zephyr commit appears in multiple places; make sure you update them all.
 
-The Zephyr fork in |NCS| (``sdk-zephyr``) contains all commits from the upstream Zephyr repository up to and including ``35264cc214fd``, plus some |NCS| specific additions.
+The Zephyr fork in |NCS| (``sdk-zephyr``) contains all commits from the upstream Zephyr repository up to and including ``856fd652dab9``, plus some |NCS| specific additions.
```

Comment:
The change is also addressed here: https://github.com/nrfconnect/sdk-nrf/pull/3682/files

Reply:
Thanks, and that's now overridden by a subsequent upmerge :) reverted this bit.